Tests/StencilTests/LexerSpec.swift

```diff
@@ -126,6 +126,17 @@ final class LexerTests: XCTestCase {
     }
   }
 
+  func testCombiningDiaeresis() throws {
+    let path = Path(#file as String) + ".."  + "fixtures" + "combining-diaeresis.html"
```

Comment:
can we maybe just inline the content of the file here? fixtures are really used to test logic specific to template inheritance and such stuff that requires files

Reply:
I wasn't sure how special Unicode symbols might affect editors and build tools, but at least Xcode 11.2.1 and its toolchain seem to be fine with it. :)